helmfile/charts/networkpolicy-common/templates/calico-accountant/allow-calico-accountant.yaml

```diff
@@ -0,0 +1,55 @@
+{{ if .Values.calicoAccountant.enabled }}
+kind: NetworkPolicy
+apiVersion: networking.k8s.io/v1
+metadata:
+  name: allow-calico-accountant
+spec:
+  podSelector:
+    matchLabels:
+      app: calico-accountant
+  policyTypes:
+    - Egress
+    - Ingress
+  ingress:
+    - ports:
```

Comment:
Do calico account have this port open? I do not really see that dns traffic will go to calico accountant.

Reply:
I have removed the policy for calico-accountant